tests/test_makefile_checks.py

```
@@ -0,0 +1,74 @@
+
+    Returns
+    -------
+    subprocess.CompletedProcess[str]
+        Completed make process with captured output.
+
+    Side Effects
+    ------------
+    Starts a subprocess in the repository root.
+    """
+    return subprocess.run(  # noqa: S603,S607
+        ["make", *args],
```

Ruff (0.15.11)

[error] Starting a process with a partial executable path

(S607)